src/Adapter/Country/ValueObject/Country.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Adapter\Country\ValueObject;
+
+use PrestaShop\PrestaShop\Core\Domain\Country\Exception\CountryConstraintException;
+
+class Country
+{
+    /** @var int */
+    private $id;
+    /** @var string */
+    private $name;
+
+    /**
+     * Country constructor.
```

Comment:
rm this 'Country constructor.' I suggest you configuring your storm file tpl to remove this, as its useless

Reply:
nice :slightly_smiling_face:  Do you know where exactly possible to do that is storm settings. Searched for that but not so easy to find how..

Reply:
changed it to PS standard as you guys have more experience with PS even though one liner looks better for me

Reply:
Thanks Julius for guiding to find: File and Code Templates -> PHP Constructor